<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtMouseMoverClass</class>
 <widget class="QMainWindow" name="qtMouseMoverClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>600</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>600</height>
   </size>
  </property>
  <property name="tabletTracking">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>Project Neutro</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QTabWidget" name="tabWidget">
      <property name="font">
       <font>
        <family>Noto Sans</family>
        <pointsize>10</pointsize>
       </font>
      </property>
      <property name="tabPosition">
       <enum>QTabWidget::North</enum>
      </property>
      <property name="tabShape">
       <enum>QTabWidget::Rounded</enum>
      </property>
      <property name="currentIndex">
       <number>1</number>
      </property>
      <widget class="QWidget" name="main_tab">
       <attribute name="title">
        <string>Main</string>
       </attribute>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QLabel" name="label_6">
          <property name="font">
           <font>
            <family>Noto Sans</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="text">
           <string>Status</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QLabel" name="status_lable">
          <property name="font">
           <font>
            <family>Noto Sans</family>
            <pointsize>16</pointsize>
           </font>
          </property>
          <property name="layoutDirection">
           <enum>Qt::LeftToRight</enum>
          </property>
          <property name="text">
           <string>Stopped</string>
          </property>
          <property name="textFormat">
           <enum>Qt::AutoText</enum>
          </property>
          <property name="scaledContents">
           <bool>true</bool>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="wordWrap">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="1" column="0" colspan="2">
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
        <item row="2" column="0">
         <widget class="QPushButton" name="stop_btn">
          <property name="enabled">
           <bool>false</bool>
          </property>
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Noto Sans</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="tabletTracking">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Stop</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QPushButton" name="start_btn">
          <property name="minimumSize">
           <size>
            <width>0</width>
            <height>50</height>
           </size>
          </property>
          <property name="font">
           <font>
            <family>Noto Sans</family>
            <pointsize>10</pointsize>
           </font>
          </property>
          <property name="tabletTracking">
           <bool>true</bool>
          </property>
          <property name="text">
           <string>Start</string>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
      <widget class="QWidget" name="setting_tab">
       <attribute name="title">
        <string>Settings</string>
       </attribute>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <widget class="QGroupBox" name="groupBox">
          <property name="title">
           <string>Main Settings</string>
          </property>
          <layout class="QGridLayout" name="gridLayout">
           <item row="4" column="0" colspan="3">
            <widget class="QPushButton" name="about_btn">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="tabletTracking">
              <bool>true</bool>
             </property>
             <property name="text">
              <string>About</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QLabel" name="second_lable">
             <property name="text">
              <string notr="true">0</string>
             </property>
             <property name="textFormat">
              <enum>Qt::PlainText</enum>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QSlider" name="timer_slider">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>20</height>
              </size>
             </property>
             <property name="tabletTracking">
              <bool>true</bool>
             </property>
             <property name="toolTip">
              <string>Drag the slider to the corresponding seconds you would like to pause the application. Max is one minitue</string>
             </property>
             <property name="maximum">
              <number>60</number>
             </property>
             <property name="pageStep">
              <number>1</number>
             </property>
             <property name="value">
              <number>30</number>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="invertedAppearance">
              <bool>false</bool>
             </property>
             <property name="invertedControls">
              <bool>false</bool>
             </property>
             <property name="tickPosition">
              <enum>QSlider::NoTicks</enum>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QCheckBox" name="tabbing_check">
             <property name="toolTip">
              <string>If you would like to let the program alt tab to pretend to switch appliction check this box. It will do it at random intervals. </string>
             </property>
             <property name="text">
              <string>Allow Tabbing</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Pause Timer</string>
             </property>
            </widget>
           </item>
           <item row="3" column="1" colspan="2">
            <widget class="QPushButton" name="defult_main_btn">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="tabletTracking">
              <bool>true</bool>
             </property>
             <property name="toolTip">
              <string/>
             </property>
             <property name="text">
              <string>Default</string>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QPushButton" name="updates_btn">
             <property name="enabled">
              <bool>false</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="tabletTracking">
              <bool>true</bool>
             </property>
             <property name="toolTip">
              <string>Check for any latest updates. This will check github to see if this version is the latest. </string>
             </property>
             <property name="text">
              <string>Updates</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QCheckBox" name="Minimise_check">
             <property name="toolTip">
              <string>Minimise this appliction to the system tray so its hidden. Keyboard shortcuts will still work. </string>
             </property>
             <property name="text">
              <string>Minimise to sytem tray</string>
             </property>
             <property name="checked">
              <bool>true</bool>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <widget class="QGroupBox" name="groupBox_2">
          <property name="title">
           <string>HotKeys</string>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="1" column="0">
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Resume Hot Key</string>
             </property>
            </widget>
           </item>
           <item row="0" column="0">
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Pause Hot Key</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QKeySequenceEdit" name="pause_hotkey">
             <property name="tabletTracking">
              <bool>true</bool>
             </property>
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Chose what you would like your pause hotkey to be the defualt is ALT + P&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="keySequence">
              <string>Alt+P</string>
             </property>
             <property name="clearButtonEnabled">
              <bool>true</bool>
             </property>
             <property name="maximumSequenceLength">
              <longLong>2</longLong>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QKeySequenceEdit" name="Resume_hotkey">
             <property name="tabletTracking">
              <bool>true</bool>
             </property>
             <property name="toolTip">
              <string>Chose what you would like your resume hotkey to be the defualt is ALT + R</string>
             </property>
             <property name="keySequence">
              <string>Alt+R</string>
             </property>
            </widget>
           </item>
           <item row="2" column="0" colspan="2">
            <widget class="QPushButton" name="defult_hotkey_btn">
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>50</height>
              </size>
             </property>
             <property name="tabletTracking">
              <bool>true</bool>
             </property>
             <property name="toolTip">
              <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;Set the value back to system defualt.&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
             </property>
             <property name="text">
              <string>Default</string>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QPushButton" name="apply_btn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="tabletTracking">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Apply</string>
            </property>
            <property name="autoDefault">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="remove_btn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="tabletTracking">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Discard</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="close_btn">
            <property name="minimumSize">
             <size>
              <width>0</width>
              <height>50</height>
             </size>
            </property>
            <property name="tabletTracking">
             <bool>true</bool>
            </property>
            <property name="text">
             <string>Close</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
